.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,31 @@
+name: build_image
+
+on:
+  pull_request:
+    types:
+      - 'synchronize'
+
+env:
+  PR_NUMBER: ${{ github.event.pull_request.number }}
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
```

Comment:
Buildx, typo?

Reply:
Nope it's an actual plugin name - https://github.com/docker/buildx